src/iris/bin/sync_targets.py

```diff
@@ -105,11 +105,14 @@ def prune_target(engine, target_name, target_type):
 
     if target_type == 'team':
         try:
-            engine.execute('''DELETE FROM `oncall_team` WHERE `target_id` = (SELECT `target`.`id` FROM `target` JOIN `target_type` ON `target`.`type_id` = `target_type`.`id` WHERE `target_type`.`name` = %s AND `target`.`name` = %s)''', target_type, target_name)
+            # rename team to prevent namespace conflict but preserve the ability to reactivate it in the future
+            new_name = str(uuid.uuid1())
```

Comment:
Might be better to use `uuid.uuid4()` instead, as that generates the UUID randomly rather than using a combination of the hostname + timestamp and is afaik more idiomatic.